packages/client/src/Components/Channel.jsx

```diff
@@ -103,6 +103,8 @@ class Channel extends Component {
             role={state.selectedRole}
             timestamp="2020-09-18T05:41:17.349Z"
             title="lorem ipsum do lor sit amet lorem ipsum do lor sit amet lorem ipsum do lor sit amet lorem ipsum do lor sit amet lorem ipsum do lor sit amet lorem ipsum do lor sit amet"
+            privateMessage="This is an example of a private message."
```

Comment:
you should not be hard-coding data into the actual component.  Data objects like this should be either provided in tests, or taken from the database.

Note that the existing code (from line 108 onwards) uses a `ChannelMessagesList` element to render the series of `messages` entries.